support.php

```diff
@@ -65,8 +65,8 @@
     </a>
 
     <a class="app" href="https://elementaryos.stackexchange.com/questions/tagged/scratch" target="_blank" rel="noopener">
-        <img width="64" height="64" src="images/icons/apps/64/accessories-text-editor.svg" alt="Text Editor"/>
-        <span>Scratch</span>
+        <img width="64" height="64" src="images/icons/apps/64/accessories-text-editor.svg" alt="Code"/>
```

Comment:
And also, I couldn't find Code's icon. Do you know where it is?

Reply:
Hm, I believe it's shipped in the Code repo. I'm not sure what the best way to include it here is.

Reply:
I see.
How about adding the icon in the Code repo to elementary Icons repo which links to website repo?